tests/plugins/colorbutton/manual/highlightedcolor.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.8.0, bug, 1008
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, colorbutton, colordialog, format
```

Comment:
I think `colordialog` is not needed here. You may also remove `format` plugin (I assume it is to have `h1` in the editor so the text is bigger and more readable - but you can also achieve this using [addCss](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR-method-addCss) and just change the `font-size`).